index.html

```diff
@@ -214,5 +214,92 @@ <h2 class="skill-title">Skills</h2>
         </li>
       </ul>
     </section>
+
+    <!-- Contact form and Footer Section -->
+
+    <footer class="footer-container" id="footer-container-id">
+      <div class="footer-form">
+        <h1 class="foot-title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h1>
+        <form
+          action="https://formspree.io/f/xbjewgnl"
+          method="post"
+          class="contact-form-input"
+        >
+          <ul class="form-input">
+            <li class="input-name">
+              <input
+                type="text"
+                name="name"
+                id="name"
+                maxlength="30"
+                placeholder="Full name"
+                required
+              />
+            </li>
+            <li class="input-email">
+              <input
+                type="email"
+                name="email"
+                id="email"
+                placeholder="Email address"
+                required
+              />
+            </li>
+            <li class="input-text">
+              <textarea
+                name="text_area"
+                id="text_area"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                placeholder="Hey I’d like to connect to..."
+                required
+              ></textarea>
+            </li>
+            <li class="submit-button">
+              <button type="submit">Get in touch</button>
```

Comment:
- [ ] Kindly add button interaction to this button so that it would take Figma's button types.
![image](https://user-images.githubusercontent.com/86200356/235657358-5e5f0633-44a8-408f-8002-0fd958ce9eb8.png)